emr-oss/src/main/java/com/aliyun/ms/MetaClient.java

```diff
@@ -43,27 +52,82 @@ static private String trySend(String host, String url) {
     }
   }
 
+  static private String trySendV2(String metaItem) {

```

Comment:
Could you rename `trySendV2` to something like `retrieveMetadataFromECS`. Apply same change to `trySend`